src/Development/IDE/Core/Shake.hs

```diff
@@ -854,8 +895,8 @@ uses key files = map (\(A value) -> currentValue value) <$> apply (map (Q . (key
 usesWithStale :: IdeRule k v
     => k -> [NormalizedFilePath] -> Action [Maybe (v, PositionMapping)]
 usesWithStale key files = do
-    values <- map (\(A value) -> value) <$> apply (map (Q . (key,)) files)
-    zipWithM lastValue files values
+    _ <- apply (map (Q . (key,)) files)
+    mapM (lastValue key) files
```

Comment:
does this change the semantics of `usesWithStale`?

Reply:
No,  I don't think so